parser/parser.y

```diff
@@ -2257,6 +2259,16 @@ FunctionCallNonKeyword:
 	{
 		$$ = &ast.FuncCallExpr{FnName: model.NewCIStr($1.(string)), Args: $3.([]ast.ExprNode)}
 	}
+|	"POW" '(' Expression ',' Expression ')'
```

Comment:
Please add some unit tests in parser/parser_test.go TestBuiltin.
https://github.com/pingcap/tidb/blob/master/parser/parser_test.go#L329